web/src/engine/websites/templates/VTheme.ts

```diff
@@ -87,8 +89,23 @@ export class VTheme extends DecoratableMangaScraper {
             });
     }
 
+    private ToNumber(text: string): number {
+        return /(\d+)/.test(text) ? parseInt(text.match(/(\d+)/).at(1)) : Number.POSITIVE_INFINITY;
+    }
+
     public override async FetchPages(chapter: Chapter): Promise<Page[]> {
         const { images } = await FetchNextJS<HydratedPages>(new Request(new URL(chapter.Identifier, this.URI)), data => 'images' in data);
+        if (this.useAlternativeSorting) {
```

Comment:
Consider to always apply sorting instead of introducing an additional flag.

Reply:
Every websites sort first according to `order`. For two of them there is that custom sorting right **after** : Hivetoon got it **disabled** (its a env flag), on Magustoon its **forced**.

I'd rather just include mandatory sorting in decorator, then override `FetchChapters` on Magustoon to sort pages like the websites wants. I feel its safer to follow the code than just "custom sort will work everywhere".
I have a feeling it may move some pages to the wrong place.

```ts
const pages = await super.Fetchapters.call(this, chapter);
return pages.sort( ....custom magus sorting)
```

Reply:
Ofc if using CSS or JS we dont have to care about that